index.html

```diff
@@ -3,47 +3,229 @@
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <title>Week 1: HTML Tables</title><!-- Latest compiled and minified CSS -->
+  <title>State reserves</title><!-- Latest compiled and minified CSS -->
   <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
   <style>
+    .section { padding: 1em 0; }
     .header {
-      text-align: center;
+      background: linear-gradient(to bottom, #337ab7 0%, #135a97 100%);
+      color: white;
+      padding: 1em 0 2em;
+    }
+    #data-table td {
+      width: 20%;
+    }
+    .bar {
+      height: 20px;
+      background: #337ab7;
     }
   </style>
 </head>
-<body>
+     
   <div class="container">
     <div class="header">
```

Comment:
Note that in my version I've now placed the container div inside the header div. I did this because the container's role is to specify a maximum width that elements can be, and if the page is bigger, to center them on the page. The header now has that nice background gradient. But if it's insider the container, it can only be as wide as the container. By placing the container inside the header, the header background stretches full width, while the h1 and p are still positioned correctly.